### ISSUE TITLE
unable to publish migration

### ISSUE THREAD

**Issue by Jonnx** [RhysLees/jetstream-user-timezone#4]
following the setup steps, the migration publish step fails

```bash
 php artisan vendor:publish --tag="jetstream-user-timezone-migrations"

   INFO  Publishing [jetstream-user-timezone-migrations] assets.  

   ERROR  Can't locate path: <vendor/rhyslees/jetstream-user-timezone/src/../database/migrations/add_timezone_field_to_users_table.php.php.stub>.  
```

**Comment by Jonnx** [comment]
can be manually copy and pasted by running the following command in the root of the project

```
cp ./vendor/rhyslees/jetstream-user-timezone/database/migrations/add_timezone_field_to_users_table.php.stub ./database/migrations/add_timezone_field_to_users_table.php
```

ensure to rename the migration file by prefixing the date to the file name to ensure the migration order is preserved:

```
YYYY_MM_DD_NNNNN_add_timezone_field_to_users_table.php
```

**Comment by RhysLees** [comment]
> can be manually copy and pasted by running the following command in the root of the project
> 
> ```
> cp ./vendor/rhyslees/jetstream-user-timezone/database/migrations/add_timezone_field_to_users_table.php.stub ./database/migrations/add_timezone_field_to_users_table.php
> ```
> 
> ensure to rename the migration file by prefixing the date to the file name to ensure the migration order is preserved:
> 
> ```
> YYYY_MM_DD_NNNNN_add_timezone_field_to_users_table.php
> ```

Thanks, it seemed to be a mistake in calling the migration file. This is now fixed in 1.0.1 😀
